src/components/form/PrintPDF.js

```diff
@@ -103,13 +109,13 @@ export default class PrintPDF extends  React.Component
     }
 
     //recoveries
-    var recoveryDescription = doc.setFontSize(8).splitTextToSize(recovery_description, 90)
+    var recoveryDescription = createParagraph(doc, recovery_description, 90)
 
     doc.setFontSize(10).text(170, 185, recoveries, 'center');
-    doc.setFontSize(8).setTextColor("#808080").text(175,210, recoveryDescription, 'center');
+    doc.setFontSize(8).setTextColor("#808080").text(172,210, recoveryDescription, 'center');
```

Comment:
I'm seeing this style duplicated in line 94, could we make this a function like we have createTitle? Perhaps createBody or createBodySmall?